blendergltf.py

```diff
@@ -250,7 +250,12 @@ def export_buffer(self, settings):
         if settings['buffers_embed_data']:
             uri = 'data:text/plain;base64,' + base64.b64encode(data).decode('ascii')
         else:
-            uri = bpy.path.clean_name(self.name) + '.bin'
+            if settings['buffers_combine_data']:
+                parts = os.path.splitext(os.path.basename(settings['filepath']))
```

Comment:
That would make the handling of other files, such as shaders and textures, inconsistent with buffers. While a little odd, I think the way things are split up with regards to file handling will do for now. I think in this case just adding another setting will work well enough.